mongomock/store.py

```diff
@@ -104,3 +107,51 @@ def __len__(self):
     def documents(self):
         for doc in six.itervalues(self._documents):
             yield doc
+
+    def remove_expired_documents(self):
+        for index in six.itervalues(self.indexes):
+            if index.get('expireAfterSeconds') is not None:
+                self._expire_documents(index)
+
+    def _expire_documents(self, index):
+        # Ignore non-integer values
+        try:
+            expiry = int(index['expireAfterSeconds'])
+        except ValueError:
+            return
+
+        # Ignore commpound keys
+        if len(index['key']) > 1:
+            return
+
+        # "key" structure = list of (field name, direction) tuples
+        ttl_field_name = index['key'][0][0]
+        expired_ids = {
+            doc['_id'] for doc in six.itervalues(self._documents)
+            if self._value_meets_expiry(doc.get(ttl_field_name), expiry)
+        }
+
+        for exp_id in expired_ids:
+            del self[exp_id]
+
+    def _value_meets_expiry(self, val, expiry):
+        val_to_compare = _get_dt_from_value(val)
+        try:
+            return (mongomock.utcnow() - val_to_compare).total_seconds() >= expiry
+        except TypeError:
+            return False
+
+
+def _get_dt_from_value(val):
```

Comment:
please rename this function to `_get_min_datetime_from_value` or just `_get_min_datetime`